e2e/testsuite/tx.go

```diff
@@ -70,6 +111,7 @@ func (s *E2ETestSuite) BroadcastMessages(ctx context.Context, chain ibc.Chain, u
 	s.Require().NoError(err)
 
 	chainA, chainB := s.GetChains()
```

Comment:
ah, I see we were already waiting for chainB , I don't think we even need to do this here.